jaxrs/src/main/java/com/cerner/beadledom/jaxrs/exceptionmappers/FailureExceptionMapper.java

```diff
@@ -0,0 +1,68 @@
+package com.cerner.beadledom.jaxrs.exceptionmappers;
```

Comment:
Packages should generally be singular. I think this could either be `jaxrs.exceptionmapper` or simply `jaxrs.exception`.

Reply:
Would you https://github.com/cerner/beadledom/pull/45/commits/3b104b966f9a41e59aa0e0f59a2c3a14a1186db4